lib/modules/hosts/wikipedia.js

```diff
@@ -55,6 +55,8 @@ export default {
 		$wikiData.find('.metadata, .hatnote, .mw-editsection, .reference, .references').remove();
 		// Make all relative links direct
 		$wikiData.find('a[href^="/"]').attr('href', (i, old) => `https://${language}.wikipedia.org${old}`);
+		// Prepend wikipedia logo to make clear it is a wikipedia article
+		$wikiData.prepend('<img src="https://en.wikipedia.org/static/favicon/wikipedia.ico" alt="Hosted on wikipedia" title="Hosted on wikipedia">')
```

Comment:
This does something differently though and site attribution in that place wouldn't work for most other expandos,